src/target_intacct/statistical_journal.py

```diff
@@ -62,6 +60,8 @@ def load_statistical_journal_entries(
         "employeeid",
         "Capacity",
         "BudgetedBillable",
+        "CapacityNo",
```

Comment:
CapacityNo and BudgetedBillableNo are the account numbers for Capacity and BudgetedBillable, I implemented them to replace the hardcoded values from before. CapacityNo is always 98050 and BudgetedBillableNo is always 98051, Capacity and BudgetedBillable are the values that the journal focuses on (ex: Capacity = 8 and BudgetedBillable = 5.8). One can't be calculated from the other, the No columns are just needed to upload the data.

Reply:
Thanks. Even the concept of `Capacity` and `BudgetedBillable` is something unique to Fresh, right?

What does this data look like from the Intacct point of view outside of Fresh Consulting? We have a `LineItemType`, a `LineItemAccountNumber`, and a `LineItemValue` or something like that?

Here are we creating two Intacct Statistical Journal entries for every input row? A more generalized approach would be to create one Intacct Statistical Journal entry for each one input row?

Ideally the Intacct Target is built from Intacct's perspective with all the functionality needed to support syncing Statistical Journal data into Intacct. Once that's done though we should be able to change what data we store in the Intacct Statistical Journal and what we call it and how much of it we store *without further changes to the Intacct Target*. How do we get there? 

We want an Intacct Target that will allow us to store the daily GBP to Euro exchange rate or the Seattle Sounders box scores in the Intacct Statistical Journal in the future without any additional changes to this repo. Can that be done?

Reply:
Yes Capacity and BudgetedBillable  are purely Fresh concepts.

Yes, on the Intacct side it's very similar to that.  Capacity and BudgetedBillable are statistical accounts and when we create each line of the journal we assign the value to  `AMOUNT` and the account number that we want it related to to `ACCOUNTNO` ex: for Capacity it could be `AMOUNT ` = 8 and `ACCOUNTNO` = 98050. 

We have to create two entries for every row in BigQuery to maintain the formatting of the Hours_Per_Week_Denominator journal, but we can definitely generalize it more. I appreciate you bringing that up and I'm sorry for not fully grasping how you wanted it implemented before. Instead of going off of the account names in the Target like it is currently implemented, we can support as many different accounts as indicated by the input, with the values listed as "AmountValue1", "AmountValue2", etc and the Accounts being listed as "AccountNO1", AccountNO2", etc or something similar. Would that align more with the generalization that you're envisioning?

Reply:
The latest commit contains changes that allow for any account number and amount. The Amount and AccountNo are associated by whatever follows the "Amount" or "AccountNo", for example in my testing I had Amount3 - AccountNo3 and AmountClientHours - AccountNoClientHours. These are the results from using four accounts for one user .
![image](https://github.com/FreshConsulting/target-intacct/assets/129103234/f6687e80-fbcb-4a26-84cd-e95ce25d5eed)
If this looks right to you, the next step would be changing the Capactity, CapacityNo, BudgetedBillable, and BudgetedBillableNo columns in the query to something like Amount1, AmountCapacity, or Amount98051